scout/server/blueprints/cases/views.py

```diff
@@ -69,13 +74,15 @@ def case(institute_id, case_name):
         return redirect(request.referrer)
 
     data = controllers.case(store, institute_obj, case_obj)
+
     return dict(
         institute=institute_obj,
         case=case_obj,
         mme_nodes=matchmaker.connected_nodes,
         tissue_types=SAMPLE_SOURCE,
         gens_info=gens.connection_settings(case_obj.get("genome_build")),
         display_rerunner=rerunner.connection_settings.get("display", False),
+        path_exists=path_exists,
```

Comment:
I see that you are passing the function to the template in this dictionary. I don't think this is super standard practice to add custom jinja filters to the web pages. 

Check the docs regarding custom filter in Jinja --> https://jinja.palletsprojects.com/en/3.0.x/api/#custom-filters

Without passing functions around and in between macros, you could define an app template filter [in here](https://github.com/Clinical-Genomics/scout/blob/f8e043aa07b458c051c9660cef2dfba73953da1f/scout/server/app.py#L167)

This filter would be your `path_exists() `function, for instance.
Then you can invoke it from the template like this: `cases.coverage_qc_report|path_exists` 

This other approach looks easier to me..

Reply:
Hm, but I dont quite want the return value filtered out. However there are custom tests as well - that would fit very nicely. I'll give it a go!

Reply:
Mm, but a filter that returns true/false is no longer really a filter, rather a test. It looks good! 😸